dashboard/src/components/docs/Moderation.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <header>
+      <h1>Moderation</h1>
+      <p>
+        Moderation in Zeppelin is multi-layered. On top of typical actions such
+        as warning, muting, kicking, and banning, Zeppelin allows moderators to
+        utilise flags; create alerts; set thresholds; and act as others.
+      </p>
+      <p>
+        This guide explains the options available in the
+        <router-link to="/docs/plugins/modactions"> Mod actions</router-link>
+        plugin. To best use this guide, read the default configuration of the
+        Mod actions plugin alongside this. This plugin does
+        <strong>not</strong> cover muting members, please see the
+        <router-link to="/docs/plugins/mutes">Mutes</router-link> plugin.
+      </p>
+      <p>
+        Please ensure you understand how
+        <router-link to="/docs/plugins/plugin-configuration">plugin
+          configuration</router-link> and
+        <router-link to="/docs/configuration/permissions">plugin
+          permissions </router-link>work before reading this guide since
+        the configs defined here rely on your these concepts.
+      </p>
+    </header>
+
+    <h2>Moderation Commands</h2>
+    <p>
+      So that your moderators may use Zeppelin moderation, you must define the
+      moderator role Id in the config, assign it a level (50), and enable the
+      Mod actions plugin.
+      <CodeBlock code-lang="yaml" trim="start">
+        levels:
+          "PRETEND-ROLE-ID": 50 # Mod
+
+        plugins:
+          mod_actions: {}
+      </CodeBlock>
+    </p>
+    <p>
+      Each moderation command has a permission attached to it, so if your
+      server has a hierarchical structure then you will be able to scope
+      these permissions by referencing the plugins permissions page.
+    </p>
+
+    <h2>Sanction Notifications</h2>
+    <p>These config options define how Zeppelin will interact with the
+      members it sanctions (warns, kicks, bans).</p>
+
+    <h3>DM Values</h3>
+    <p>
+      The values <code>dm_on_warn</code>, <code>dm_on_kick</code>, and
+      <code>dm_on_ban</code> determine whether a member will be notified of
+      their sanctions through DMs. Ignoring privacy settings, setting these to
+      <code>true</code> will notify the member. Temporary banning uses the
+      ban configuration.
+    </p>
+
+    <h3>Channel Values</h3>
+    <p>
+      An alternative way to notify members about sanctions is through
+      mentioning them in a message sent in a channel. To enable this feature,
+      set <code>message_on_warn</code>, <code>message_on_kick</code>, and
+      and <code>message_on_ban</code> to true (enabling them), then assign a
```

Comment:
This is already implied by the start of the sentence ("To enable this feature")
```suggestion
      and <code>message_on_ban</code> to true, then assign a
```